x-pack/plugin/spatial/src/test/java/org/elasticsearch/xpack/spatial/util/ShapeUtilTests.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.spatial.util;
+
+import org.apache.lucene.geo.XShapeTestUtil;
+import org.apache.lucene.geo.XYPolygon;
+import org.apache.lucene.geo.XYRectangle;
+import org.elasticsearch.test.ESTestCase;
+
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.lessThan;
+
+public class ShapeUtilTests extends ESTestCase {
+    public void testBox() {
```

Comment:
So this test class has a few pretty arbitrary tests in it. This is because there was no existing test class for ShapeTestUtils, and I wanted at least some test of the new `area()` function, and added a couple of basic tests just make a start on perhaps adding more test coverage later. Since we're testing a test-only class the need for tests is not that high, but I think it still has value, and also is very useful when wanting to debug specific test methods.